north/src/manifest.rs

```diff
@@ -221,6 +313,28 @@ impl Manifest {
         })
         .await
     }
+
+    /// Returns a list of tuples `(target, Resource)`
+    pub fn resources(&self) -> Vec<(&std::path::Path, &Resource)> {
+        self.mounts
+            .iter()
+            .filter_map(|e| match &e.source {
+                MountSource::Resource(r) => Some((e.target.as_path(), r)),
+                _ => None,
+            })
+            .collect()
+    }
+
+    /// Returns a list of tuples `(target, DirMount)`
+    pub fn mounts(&self) -> Vec<(&std::path::Path, &DirMount)> {
+        self.mounts
+            .iter()
+            .filter_map(|e| match &e.source {
+                MountSource::Directory(m) => Some((e.target.as_path(), m)),
+                _ => None,
+            })
+            .collect()
```

Comment:
Changing `mounts` to be a enum this two methods are obsolete. The user of a `Manifest` then do not need to do a extra call to extract the lists.

Reply:
this one is just for convenience and remove the match expressions in process.rs and mock.rs